functions/get-token-from.js

```diff
@@ -0,0 +1,8 @@
+const getTokenFrom = (req) => {
+  const authorization = req.header('authorization');
+  if (authorization && authorization.toLowerCase().startsWith('bearer ')) {
```

Comment:
The method `toLowerCase` will change the token received

Reply:
JWT generate a string with Uppercase y Lowercase letters, how say gonza, it`s changing the token.

Reply:
No! It will not modify token because toLowerCase is used just to verify if it starts with 'bearer'.